lib/rubycritic/source_locator.rb

```diff
@@ -7,6 +7,7 @@ module RubyCritic
   class SourceLocator
     RUBY_EXTENSION = '.rb'.freeze
```

Comment:
This will still skip files ending in `.rake` or `.thor`. Maybe it would be better to add a `ruby_extensions` key to the .rubycritic.yml config file and source that value here as an array and give it a good default like `%w(.rb .rake .thor)`. That way anyone wishing to check other file extensions that also don't use a shebang, they're all set, just add it to the config. 

Looks goo other wise :)

Reply:
@jtgrenz Good point. Thanks for looking into this!

Reply:
Sorry for the long delay in getting back to this. I've added a commit to make the ruby extensions configurable. Let me know if there's anything else we could improve.